src/main/resources/queries/rumbleML/sample-ml-data-nested2.json

```diff
@@ -0,0 +1,6 @@
+{"label": 0, "object": {"intarray": [1,2,3], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
+{"label": 1, "object": {"intarray": [4,5,6], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
+{"label": 2, "object": {"intarray": [7,8,9], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
+{"label": 3, "object": {"intarray": [1,4,7], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
+{"label": 4, "object": {"intarray": [2,5,8], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
+{"label": 5, "object": {"intarray": [3,6,9], "stringarray": ["abc"]}, "array": [{"string": "a", "decimal": "3.5"}]}
```

Comment:
How about also varying the size of the arrays rather than keep them constant? (In my understanding, this should work as well in dataframes, as array sizes are open).

Reply:
Very good suggestion! Added a new test for that